### PR TITLE
Social icons, default content block placement on homepage.

### PR DESCRIPTION
**Pull request by benjy**
Note, i set the icons to only appear on <front> but we don't have a homepage configured yet so you have to go and set homepage to /node/1 to see the icons.

### CODE REVIEW

File: agov/modules/custom/agov_default_content/config/optional/block.block.stayuptodate.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+langcode: en
```
Comment (Sam152) [inline agov/modules/custom/agov_default_content/config/optional/block.block.stayuptodate.yml]: I think the block placement should be part of the config that is installed with the 'agov_social_icons' module. On a fresh prod install, you wouldn't expect to have to place a block to use the feature.

File: agov/modules/custom/agov_default_content/config/optional/block.block.stayuptodate.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  label_display: visible
+  cache:
+    max_age: -1
+  status: true
+  info: ''
+  view_mode: full
+visibility:
+  request_path:
+    id: request_path
+    pages: '<front>'
```
Comment (Sam152) [inline agov/modules/custom/agov_default_content/config/optional/block.block.stayuptodate.yml]: On 3.x these are in the sidebar for all pages.

File: agov/modules/custom/agov_social_icons/agov_social_icons.info.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name: aGov Social Icons
+type: module
+description: 'Provide a social icon block type.'
+package: aGov
+core: '8.x'
+
+dependencies:
+  - block
+  - link
+  - linkicon
```
Comment (Sam152) [inline agov/modules/custom/agov_social_icons/agov_social_icons.info.yml]: Is this missing from the make file?

Comment (benjy) [inline agov/modules/custom/agov_social_icons/agov_social_icons.info.yml]: Added.

File: agov/modules/custom/agov_default_content/config/optional/block.block.stayuptodate.yml (hunk 2 above)
Comment (benjy) [inline agov/modules/custom/agov_default_content/config/optional/block.block.stayuptodate.yml]: removed visibility rules.

File: agov/modules/custom/agov_default_content/config/optional/block.block.stayuptodate.yml (hunk 1 above)
Comment (benjy) [inline agov/modules/custom/agov_default_content/config/optional/block.block.stayuptodate.yml]: Lets leave as is for now, easy to move back in future.

**Pull request merged**